src/applications/vaos/new-appointment/newAppointmentFlow.js

```diff
@@ -482,6 +495,12 @@ export default function getNewAppointmentFlow(state) {
         ? 'location'
         : '/new-appointment/va-facility-2',
     },
+    selectProvider: {
+      ...flow.selectProvider,
+      url: featureBreadcrumbUrlUpdate
```

Comment:
Isn't breadcrumb url update already fully released and pending removal? Why are we still adding code that depends on it?